widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx

```diff
@@ -141,7 +141,7 @@ export function WalletList(props: PropTypes) {
           walletType: wallet.type,
           chain,
         });
-        const conciseAddress = address ? getConciseAddress(address) : '';
+        const conciseAddress = address ? getConciseAddress(address, 7) : '';
```

Comment:
I believe the 'no-magic-numbers' ESLint rule has been deactivated in this file during previous modifications. Could you please eliminate it from the beginning of the file and introduce a constant for the number (7)?

Reply:
fixed